### ISSUE TITLE
Slack logging stacktrace when user edits message

### ISSUE THREAD

**Issue by LuckyHyena** [gbin/err#380]
When a user edits a Slack message the RTM api sends an object with `'type': 'message', 'subtype': 'message_changed'`. Err tries to parse this as an ordinary message, and prints a stacktrace to the log. I haven't seen any other bad behavior caused by this, though.

```
16:05:53 DEBUG    errbot.backends.slack     Processing slack event: {'type': 'message', 'subtype': 'message_changed', 'channel': 'C0000000S', 'ts': '1434402356.000121', 'hidden': True, 'message': {'ts': '1434402337.000118', 'text': '<@U00000003>: redacted', 'type': 'message', 'edited': {'ts': '1434402356.000000', 'user': 'U0000000Z'}, 'user': 'U0000000Z'}, 'event_ts': '1434402356.048649'}
16:05:53 DEBUG    errbot.backends.slack     Handling message from a public channel
16:05:53 ERROR    errbot.backends.slack     message event handler raised an exception
Traceback (most recent call last):
  File "/home/whiskeyriver/Code/err/errbot/backends/slack.py", line 154, in serve_once
    event_handler(message)
  File "/home/whiskeyriver/Code/err/errbot/backends/slack.py", line 208, in _message_event_handler
    msg = Message(event['text'], type_=message_type)
KeyError: 'text'
```

**Comment by WhiskeyRiver** [comment]
there's also a "message_deleted" subtype with the same problem

**Comment by zoni** [comment]
This should be fixed by PR #382. If you could give the associated branch a quick try and see if you can spot any more issues with it, that'd be great!

**Comment by gbin** [comment]
I ported #382 to head, it seems to work now
